<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OntDocumentationDialog</class>
 <widget class="QDialog" name="OntDocumentationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>RDF File:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsFileWidget" name="inputRDFFileWidget"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Export Path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="createDocumentationButton">
     <property name="text">
      <string>Create Documentation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QgsFileWidget" name="outFolderWidget">
     <property name="storageMode">
      <enum>QgsFileWidget::GetDirectory</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Namespace:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="namespaceCBox"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Create ontology documentation for data instances in this dialog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
